### DISCUSSION TITLE
Can’t use atom in filezilla(Fedora 34)

### DISCUSSION THREAD

**Discussion by yanikturgeon** [atom/atom#22585] [Q&A]
install Atom with software installer in Fedora 34.
I use Filezilla for ftp for edit files, i can’t chose Atom.
edit > settings > file editting

please help me.

**Comment by DeeDeeG** [comment]
Maybe this tutorial will help?

https://filezillapro.com/docs/v3/advanced/how-to-set-text-editor-in-three-different-ways/

Atom is usually located at `/usr/share/atom/atom`. You can set your editor as `/usr/share/atom/atom` in the Filezilla settings.

(Or for Atom Beta, Nightly or Dev editions, you can set it to: `/usr/share/atom-beta/atom`... `/usr/share/atom-nightly/atom`... `/usr/share/atom-dev/atom`.)
